openedx/README.rst

```diff
@@ -1,10 +1,13 @@
 Open edX
 --------
 
-This is the root package for Open edX.
-The intent is that all importable code from Open edX will eventually live here,
-including the code in the lms, cms, and common directories.
+This is the root package for Open edX. The intent is that all importable code
+from Open edX will eventually live here, including the code in the lms, cms,
+and common directories.
 
-Note: for now the code is not structured like this, and hence legacy code will
-continue to live in a number of different packages. All new code should be
-created in this package, and the legacy code will be moved here gradually.
+If you're adding a new Django app, place it in core/djangoapps. If you're adding
+code that's not a Django app per se but is widely useful, put it in core/lib.
```

Comment:
I've noticed that some of the code in core/lib does have Django dependencies (e.g., importing Django exceptions). What about s/that's not a Django app per se/that requires no Django models or views?